projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html

```diff
@@ -0,0 +1,78 @@
+<geoengine-sidenav-header>Raster Scaling / Unscaling</geoengine-sidenav-header>
+
+<geoengine-dialog-help>
+    <p>
+        This operator applies scaling or unscaling to a raster data.
+        <br />
+        The raster scaling operator scales/unscales the values of a raster by a given scale factor and offset. This is done by applying the
+        formulas to every pixel.
+        <br />
+        For <b> unscaling </b> the formula is: <i>p_new = p_old * slope + offset</i>. <br />
+        For <b> scaling </b> the formula is: <i>p_new = (p_old - offset) / slope</i>. <br />
+        <i>p_old</i>> and <i>p_new</i> refer to the old and new pixel values. The slope and offset values are either properties attached to
+        the input raster or a fixed value. For more information see
+        <a href="https://docs.geoengine.io/operators/rasterunscaling.html">docs.geoengine.io</a>.
+    </p>
+</geoengine-dialog-help>
+
+<form [formGroup]="form" (ngSubmit)="add()" fxLayout="column" ngNativeValidate>
+    <div fxFlex>
+        <geoengine-layer-selection [types]="inputTypes" formControlName="layer"></geoengine-layer-selection>
+        <geoengine-dialog-section-heading title="Config" subtitle="Specify the output data type"></geoengine-dialog-section-heading>
+
+        <div formGroupName="scale">
+            <mat-slide-toggle formControlName="byKey" #byKey>Use scale factor from metadata key</mat-slide-toggle>
+            <mat-form-field appearance="fill">
+                <mat-label>Scale metadata <i>domain</i></mat-label>
+                <input matInput type="text" formControlName="domain" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
```

Comment:
can you reduce the margin/padding between the domain and key? they belong together

Reply:
[1a42aea](https://github.com/geo-engine/geoengine-ui/pull/471/commits/1a42aeabc88bf4170889625b2bbe12e1eff39685)

Reply:
can you further reduce the margin like in the neighborhood aggregate?

Reply:
[f80267e](https://github.com/geo-engine/geoengine-ui/pull/471/commits/f80267e42cc90d372093a6d0480431105e1da490)